README.md

```diff
@@ -14,18 +9,48 @@ This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodu
 
 To clone using HTTPS:
 ```
-git clone https://github.com/FreeRTOS/Lab-Project-coreMQTT-Agent.git --recurse-submodules
+git clone https://github.com/FreeRTOS/coreMQTT-Agent-Demos.git --recurse-submodules
 ```
 Using SSH:
 ```
-git clone git@github.com:FreeRTOS/Lab-Project-coreMQTT-Agent.git --recurse-submodules
+git clone git@github.com:FreeRTOS/coreMQTT-Agent-Demos.git --recurse-submodules
 ```
 
 If you have downloaded the repo without using the `--recurse-submodules` argument, you need to run:
 ```
 git submodule update --init --recursive
 ```
 
+## Getting started
+The [documentation page](https://freertos.org/mqtt/mqtt-agent-demo.html) for this repository contains information on the MQTT agent and the contained demo project.  There is also a [supplemental documentation page](https://freertos.org/ota/ota-mqtt-agent-demo.html) that describes how to run an Over the Air (OTA) update agent as one of the RTOS tasks that share the same MQTT connection.
+
+## Building Demos
+This repository contains build files for the FreeRTOS Windows port - able to be run with the free [Community version of Visual Studio](https://visualstudio.microsoft.com/vs/community/) - and an ARM Cortex M model in the [QEMU hardware emulator](https://www.qemu.org/).
+
+### Visual Studio
+
+1. From the Visual Studio IDE, open the `mqtt_multitask_demo.sln` Visual Studio solution file in the `build/VisualStudio/` directory.
+
+1. Select **Build Solution** from the IDE's **Build** menu
+
+### QEMU Cortex-M3
```

Comment:
Will there be no instructions on how to run the executable in QEMU? It could be helpful for the reader to be pointed to a link for using QEMU

Reply:
I would add it if I knew, but unfortunately I have not tried running it

Reply:
Added.